common/grammar-parser.cpp

```diff
@@ -93,6 +97,17 @@ namespace grammar_parser {
         while (is_word_char(*pos)) {
             pos++;
         }
+        if (pos == src) {
+            throw std::runtime_error(std::string("expecting integer at ") + src);
```

Comment:
```suggestion
            throw std::runtime_error(std::string("expecting identifier at ") + src);
```

Reply:
Am I mistaken about these error messages being swapped? I could be reading it wrong.